Stytch/DemoApps/B2BWorkbench/ViewControllers/AuthMethodControllers/SessionsViewController.swift

```diff
@@ -70,7 +70,7 @@ final class SessionsViewController: UIViewController {
         }
         Task {
             do {
-                let parameters = StytchB2BClientSessions.ExchangeParameters(organizationID: organizationID)
+                let parameters = StytchB2BClient.StytchB2BClientSessions.ExchangeParameters(organizationID: organizationID)
```

Comment:
This should just be `StytchB2BClient.Sessions`, right?

Reply:
No, if you recall I had renamed these but I failed to name space them, then I name spaced them, but left the naming. I can still rename them again now that they are named spaced to be just "Sessions"